README.md

```diff
@@ -6,30 +6,38 @@ developing operating systems and firmware for PXE-capable platforms.
 
 It was inspired by effort required to test PC Engines apu2 platform.
 
-`./init.sh` builds container and runs it exposing port 69. There is silent
-assumption that your host doesn't have port 69 in use.
-
 Usage
 -----
 
 ```
 git clone https://github.com/3mdeb/pxe-server.git
-./init.sh
+cd pxe-server
+CLOUD_USER=<cloud-username> NFS_SRV_IP=<server-ip> ./init.sh
```

Comment:
@miczyg1 why `CLOUD_USER` is needed ? This is public repository and all components should be available for users.

Reply:
@miczyg1 doesn't look like bulletproof:

```
Password for user ‘pietrushnic’: 
--2017-08-28 21:48:07--  https://cloud.3mdeb.com/remote.php/webdav/projects/pcengines/OSimages/Debian.tar.gz
Resolving cloud.3mdeb.com (cloud.3mdeb.com)... 81.95.197.197
Connecting to cloud.3mdeb.com (cloud.3mdeb.com)|81.95.197.197|:443... connected.
HTTP request sent, awaiting response... 401 Unauthorized
Authentication selected: Basic realm="Nextcloud"
Reusing existing connection to cloud.3mdeb.com:443.
HTTP request sent, awaiting response... 404 Not Found
2017-08-28 21:48:08 ERROR 404: Not Found.

tar (child): Debian.tar.gz: Cannot open: No such file or directory
tar (child): Error is not recoverable: exiting now
tar: Child returned status 2
tar: Error is not recoverable: exiting now
Enter 3mdeb cloud user password
```

Reply:
@pietrushnic I had no idea where to store the created Debian rootfs and voyage netinst. I may just cut those out of the scripts and leave pure nfs and tftp server. But it will mean that user will have to add exports and attach exported  directories to container manually. I pushed it so it could be tried out. 

Indeed, I should ensure that download was successful.

Reply:
@miczyg1 nextcloud give ability to expose publicly visible URL, so external users can download given pacakge. Your implementation limits user base to 3mdeb employees.